src/sentry/integrations/github/client.py

```diff
@@ -427,7 +428,7 @@ def _populate_trees_process_error(self, error: ApiError, extra: dict[str, str])
         else:
             # We do not raise the exception so we can keep iterating through the repos.
             # Nevertheless, investigate the error to determine if we should abort the processing
-            logger.error("Continuing execution. Investigate: %s", error_message, extra=extra)
+            capture_message("Continuing execution. Investigate: %s", error_message, extra=extra)
```

Comment:
In the original PR it used to be this:
<img width="655" alt="image" src="https://github.com/user-attachments/assets/c12b7c0f-3269-4915-ba2c-71f028f7dc30">

I'm removing `level="warning"` since it is not accepted.